ontology/uco/observable/observable.ttl

```diff
@@ -4447,8 +4447,16 @@ observable:MobileDeviceFacet
 		[
 			sh:datatype xsd:integer ;
 			sh:maxCount "1"^^xsd:integer ;
+			sh:message "Deprecation notice: the object referenced by observable:storageCapacityInBytes will be required to be an observable:totalStorageCapacityInBytes in UCO 2.0.0."@en ;
 			sh:nodeKind sh:Literal ;
 			sh:path observable:storageCapacityInBytes ;
+			sh:severity sh:Warning ;
+		] ,
+		[
+			sh:datatype xsd:integer ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
```

Comment:
This shape enters us into a new design pattern of `sh:PropertyShape`s that are focused on just a single constraint.

The shape above for `observable:totalStorageCapacityInBytes` will catch the broad spectrum of constraints - needs to be datatype integer, a literal node kind, etc.

This shape needs to be only this:

```turtle
[
  sh:maxCount "0"^^xsd:integer ;
  sh:message "Deprecation notice: the value referenced by observable:storageCapacityInBytes will be required to be stored in observable:totalStorageCapacityInBytes in UCO 2.0.0."@en ;
  sh:severity sh:Warning ;
  sh:path observable:totalStorageCapacityInBytes ;
]
```

(Note I also adjusted the message a bit.)

The `maxCount` is the only violable constraint in this shape.  The other properties adjust the user experience for what happens when they use this deprecated property.

Reply:
@b0bkaT , apologies for my typo.  I put the wrong property into `sh:path` in my sketch; it was supposed to be `observable:storageCapacityInBytes`.  Patch pushed with correction.